tests/processor/test_tasks.py

```diff
@@ -78,62 +79,78 @@ def test_get_report_exception(self, fake_downloader):
 
         self.assertEqual(report, [])
 
-    @patch('masu.processor._tasks.download._get_report_files',
-           return_value=['file1', 'file2'])
+class ProcessReportFileTests(MasuTestCase):
+    """Test Cases for the Orchestrator object."""
+
+    @patch('masu.processor.report_processor.ReportProcessor')
+    @patch('masu.processor.report_processor.ReportProcessor.process', return_value=2)
+    def test_process_file(self, fake_processor, fake_process):
+        """Test task"""
+        request = {'report_path': '/test/path/file1.csv',
+                   'compression': 'gzip',
+                   'schema_name': 'testcustomer'}
+        _process_report_file(**request)
```

Comment:
This test is missing an assertion.

Reply:
I had a merge conflict that I think I did not handle well. I'll get it back in.

Reply:
Ended up not being from the merge conflict. They were just missing. I believe this should cover it.